_template/mapper/configmap/parse.go

```diff
@@ -23,8 +23,8 @@ import (
 
 	"k8s.io/klog/v2"
 
-	"github.com/kubeedge/mappers-go/mappers/common"
 	"github.com/kubeedge/mappers-go/mappers/Template/globals"
+	"github.com/kubeedge/mappers-go/mappers/common"
```

Comment:
It's better to leave the old for the alphabet order.

Reply:
Sorry, I didn't aware of it, it's not what I want to do. I'll do it better!